chroma-common/chroma-common.spec

```diff
@@ -1,7 +1,7 @@
 %{!?name: %define name chroma-common}
-%{?!version: %define version %(%{__python2} -c "from chroma_common import version; sys.stdout.write(version())")}
+%{?!version: %define version %(%{__python} -c "from chroma_common import version; sys.stdout.write(version())")}
```

Comment:
Was there a particular reason for switching `__python2` to `__python`?

Since we only work with Python 2.x we should be targeting the 2.x version of python.  `__python` will be changed to point at Python 3.x at some point in the future and we need to make sure we are compatible with Python 3.x before falling back to the generic `__python` macros.